model/task/task.go

```diff
@@ -107,14 +107,19 @@ type Task struct {
 	// container to run it. It only applies to tasks running in containers.
 	ContainerAllocated bool `bson:"container_allocated" json:"container_allocated"`
 
-	BuildId                 string       `bson:"build_id" json:"build_id"`
-	DistroId                string       `bson:"distro" json:"distro"`
-	Container               string       `bson:"container,omitempty" json:"container,omitempty"`
-	BuildVariant            string       `bson:"build_variant" json:"build_variant"`
-	BuildVariantDisplayName string       `bson:"build_variant_display_name" json:"-"`
-	DependsOn               []Dependency `bson:"depends_on" json:"depends_on"`
-	NumDependents           int          `bson:"num_dependents,omitempty" json:"num_dependents,omitempty"`
-	OverrideDependencies    bool         `bson:"override_dependencies,omitempty" json:"override_dependencies,omitempty"`
+	BuildId  string `bson:"build_id" json:"build_id"`
+	DistroId string `bson:"distro" json:"distro"`
```

Comment:
Done. I didn't do this earlier because I was concerned that some existing queries would break if this wasn't always populated. I _think_ I've modified all the queries that rely on distro-specific logic to check specifically host tasks at this point, so it should be safe to make this omitempty.